sdk/aws/ec2.py

```diff
@@ -2,8 +2,18 @@
 from config import config
 from sdk.tools.helpers import get_values_for_key_from_dict_of_parameters
 import logging
+import random
+import string
+import traceback
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+console_handler.setFormatter(formatter)
+logger.addHandler(console_handler)
```

Comment:
you don't need this for logging, the logs are centralized now and you can setup it by using the `.env`
just put this in there `LOG_LEVEL=INFO`